### DISCUSSION TITLE
Add "Jobs" channel to slack

### DISCUSSION THREAD

**Discussion by ssyberg** [rudderlabs/rudder-server#2777] [Feature requests]
This isn't actually an RS feature request, but wasn't sure where to suggest this. Would be great if the [rudderstack slack](https://app.slack.com/client/T01E4PJL5J7/C01E1EVHTM3) had a place to post RS adjacent jobs and contracts!

**Comment by gitcommitshow** [comment]
agree. this will be helpful.

**Comment by gitcommitshow** [comment]
✅ There is a `#jobs` channel in [RudderStack slack](https://www.rudderstack.com/join-rudderstack-slack-community/) now. Do join and share any opportunities you have for RudderStack community of Data Engineers, Data Scientists, and all other Data folks.
